be/src/testing/testing.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, MethodNotAllowedException } from '@nestjs/common';
+import { ItemService } from '../item/item.service';
+import { CreateItemInput } from 'src/item/dto/create-item.input';
+import { CategoryService } from 'src/category/category.service';
+import { CreateCategoryInput } from 'src/category/dto/create-category.input';
+import { CreateNoteInput } from 'src/note/dto/create-note.input';
+import { NoteService } from 'src/note/note.service';
+import { UserService } from 'src/user/service/user.service';
+import { CreateUserDTO } from 'src/user/dto/user.create.dto';
+import { Role } from 'src/user/roles/user.roles';
+import * as faker from 'faker';
+
+@Injectable()
+export class TestingService {
+  constructor(
+    private readonly categoryRepo: CategoryService,
+    private readonly itemRepository: ItemService,
+    private readonly noteRepo: NoteService,
+    private readonly userRepo: UserService,
+  ) {
+    if (process.env.ADD_FAKE_DATA === 'TRUE') this.AddFakeData();
+    console.log('add fake data ', process.env.ADD_FAKE_DATA);
+  }
+
+  AddFakeData() {
```

Comment:
I think overall this function could be a bit more advanced, but I guess for now this will do